utils/utils.go

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package utils
+
+// RemoveArrayDuplicate elements
+func RemoveArrayDuplicate(tagArr []string) []string {
```

Comment:
Could this be renamed to RemoveDuplicateEntries to make clear what its doing ?

Reply:
and this works for slices too right, so `tagArr` -> `slice`?